### DISCUSSION TITLE
How to scroll to a specific element in a dynamically loaded list <ul>?

### DISCUSSION THREAD

**Discussion by elnatan2604** [seleniumbase/SeleniumBase#2804] [Q&A] [answered]
I have an unordered list element with over 50 list items, but only the first 10 are loaded initially. I need to scroll for example to the 45th item when the item isn't loaded at first. for example my XPath is (//ul[@title="test"]//li[@title='45']) but I can't find it since it's not loaded initially. How can I scroll to this item and ensure it's loaded before interacting with it?
Just pointing out that I use python with my seleniumbase code.

**Comment by mdmintz** [comment, marked as answer]
There's a `seleniumbase` method: `sb.scroll_to_bottom()`, which scrolls to the bottom of the page. You might be able to use it in a Python loop so that more elements continue to appear before you reach the element that you want.

**Reply by elnatan2604** [reply]
I tried it but it scroll down to the end of the page and not to the end of the element list

**Reply by mdmintz** [reply]
Then you need to scroll to the bottom item of that list first to see more results. You can use the `sb.scroll_to(selector)` method for that. Or you can use `sb.press_down_arrow()` continuously to scroll the page down until you reach the element you need.

**Reply by elnatan2604** [reply]
I tried also the scroll_to_element and to get to the 10th <li> and then wait fot the other elements to be loaded and it didnt work as well.
